docs/advanced/integrations/jenkins.md

```diff
@@ -0,0 +1,48 @@
+With Jenkins it is possible to do scriped pipelines but also declarative pipelines. I prefere declarative pipelines. You will find more information about pipelines in jenkins at https://www.jenkins.io/doc/book/pipeline/.
+
+The result parser is included in an existing plugin called Warning NG (https://plugins.jenkins.io/warnings-ng/).
+
+Ensure that the plugin is installed, than you need to execute the scan in the shell
+```
+sh "trivy image -f json -o results.json aquasec/trivy:0.19.2"
```

Comment:
sh "trivy image -f json -o results.json nginx:1.18"
It will be more clear in the sample, using a common image